src/Microsoft.Extensions.ProjectModel.Sources/DotNet/ProjectContextExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.ProjectModel;
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    internal static class ProjectContextExtensions
+    {
+        private const string DefaultConfiguration = "Debug";
+        public static IProjectContext AsAbstract(this ProjectContext context)
```

Comment:
There was like one use case when I started writing these abstractions, but I think that use case is gone now....so I'm removing this too.